.env.example

```
@@ -9,3 +9,9 @@ TEMPLATE_REPOSITORY_NAME=repository
 TEMPLATE_REPOSITORY_OWNER=organization
 
 BRANCHES_TO_PROCESS=master|main
+
+PREPENDING_HEADER_TEMPLATE="# THIS CODE WAS AUTOGENERATED. DO NOT MODIFY THIS FILE DIRECTLY
+                            # THE SOURCE CODE LIVES IN A DIFFERENT REPOSITORY:
+                            #   - xxxxx
+                            # ACCOUNTABILITY:
+                            # STEWARD:
```

```suggestion
PREPENDING_HEADER_TEMPLATE="# THIS CODE WAS AUTOGENERATED. DO NOT MODIFY THIS FILE DIRECTLY
                            # THE SOURCE CODE LIVES IN A DIFFERENT REPOSITORY:
                            #   - xxxxx
                            # ACCOUNTABILITY:
                            # STEWARD:"
```

Just to make sure other `.env` variables following this aren't seen as part of the string

---

Nice catch!

---

But! This didn't help. It only works when everything is in one line - playing around with this a bit more.